### ISSUE TITLE
LiveDisplay "New window for this data" gearbutton broken, nullpointer to the DataProvider

### ISSUE THREAD

**Issue by ctr26** [nicost/micro-manager#65]
Hi again,

On the latest nightly build(s) of mm2-gamma the GearButton for duplicating live image windows is broken. This is due to a nullpointer to the dataprovider (and possibly other objects) in the DisplayController object. 

 ```
           System.out.println(studio_);
            System.out.println(studio_.getSnapLiveManager());
            System.out.println(studio_.getSnapLiveManager().getDisplay());
//            System.out.println(studio_.getSnapLiveManager().getDisplay().duplicate());
            System.out.println(studio_.getSnapLiveManager().getDisplay().getDisplaySettings());
            System.out.println(studio_.getSnapLiveManager().getDisplay().getDataProvider());
            System.out.println(studio_.getSnapLiveManager().getDisplay().getDisplaySettings().getAllChannelSettings());
```
Tested in IntelliJ
```
org.micromanager.internal.MMStudio@2f131c67
org.micromanager.internal.SnapLiveManager@79d9201f
org.micromanager.display.internal.displaywindow.DisplayController@f3ecb61
2019-08-23T15:25:52.592213 tid0x70000b1b2000 [IFO,App] 
[       ] java.lang.NullPointerException in Thread[AWT-EventQueue-0,6,main]
```

**Comment by nicost** [comment]
I can not reproduce this.  

I start MM with a demo-configuration, start live mode, click on the gear button, select Duplicate.  A dialog shows up (the first line "min max" in that dialog should be remove), I click OK and I get a duplicate of the live window.  Even works when live mode is active.   You must be doing something different (or be working with different code).  Please give a reproducible procedure to replicate the bug.

**Comment by ctr26** [comment]
Duplicate is fine,   "New Window for this Data". I did this on a clean download of MM on an up to date Mac.

**Comment by nicost** [comment]
Just fixed that (commit 4d6799ae0).

**Comment by ctr26** [comment]
Thanks very much!
